minesweeper.py

```diff
@@ -224,6 +219,13 @@ def flagging(event):
 )
 flag_image = PhotoImage(file=(where_this_file_is / "flag_small.png"))
 bomb_image = PhotoImage(file=(where_this_file_is / "bomb_small.png"))
+frames = [
```

Comment:
Can you rename `frames` to `gif_frames` or something? It's not immediately clear that `gif_label`, `gif_frame` and `Game.win_animation` are related.